src/managers/CursorManager.cpp

```diff
@@ -266,9 +266,6 @@ void CCursorManager::updateTheme() {
             highestScale = m->scale;
     }
 
-    if (std::round(highestScale * m_iSize) == m_sCurrentStyleInfo.size)
```

Comment:
why?

Reply:
It fixes a bug where you had to apply xcursor themes twice to have it actually be applied
also
>The git blame for these lines is for a bug fix, but I believe this was left over when CursorManager was later changed?
either way code should still work the same, just a little less efficient i guess

Reply:
oke right b